src/TypeGen/TypeGen.Core/Business/TypeService.cs

```diff
@@ -44,8 +44,15 @@ public string GetTsSimpleTypeName(Type type)
                     return "boolean";
                 case "System.String":
                     return "string";
+                case "System.SByte":
+                case "System.Byte":
+                case "System.Char":
```

Comment:
Not sure if char should be categorized as numeric type. Are you using it as a numeric type? Maybe it should translate to "string" rather than "number"? What do you think?
BTW Thanks very much for a pull request, I appreciate it!

Reply:
Byte and Char are numeric. You can code like `let foo = 0x1` in javascript/typescript. It's the developer own responsibility to keep the size of the integers in the code (i.e.: he using `10000` in `byte` type).
I have add all integer types because some developer (me) may don't want to change their models/pocos/dtos to make it compatible with TypeGen.

Reply:
Yes, you're right, you can definitely treat a char as a number.
However I think that in a more general case, it represents a text character.

So to avoid confusion, I'll map it to "string" in TypeGen.
If you're using chars as numbers, you can still annotate your properties with `TsTypeAttribute` (so that it will be generated to "number").
Or just create a NuGet package from your fork and use it instead of the NuGet gallery TypeGen release.

Thanks for the effort though, I really appreciate it!
I'll merge your changes, but will move char to "string" category - that's at least in "official" version.

Reply:
That will be fine. I'm going change this for you, then you can easily merge this pull request.

Reply:
Thanks!